app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt

```diff
@@ -0,0 +1,184 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.about
+
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.core.content.pm.PackageInfoCompat
+import androidx.fragment.app.Fragment
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import kotlinx.android.synthetic.main.fragment_about.view.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.components.lib.state.ext.consumeFrom
+import org.mozilla.fenix.BuildConfig
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.metrics
+import org.mozilla.fenix.settings.SupportUtils
+import org.mozilla.fenix.settings.about.AboutItemType.LICENSING_INFO
+import org.mozilla.fenix.settings.about.AboutItemType.PRIVACY_NOTICE
+import org.mozilla.fenix.settings.about.AboutItemType.RIGHTS
+import org.mozilla.fenix.settings.about.AboutItemType.SUPPORT
+import org.mozilla.fenix.settings.about.AboutItemType.WHATS_NEW
+import org.mozilla.fenix.share.AboutView
+import org.mozilla.fenix.whatsnew.WhatsNew
+import org.mozilla.geckoview.BuildConfig as GeckoViewBuildConfig
+
+/**
+ * Displays the logo and information about the app, including library versions.
+ */
+class AboutFragment : Fragment() {
+    private lateinit var store: AboutPageStore
+    private lateinit var appName: String
+    private lateinit var aboutView: AboutView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val rootView = inflater.inflate(R.layout.fragment_about, container, false)
+        appName = getString(R.string.app_name)
+        activity?.title = getString(R.string.preferences_about, appName)
+
+        store = AboutPageStore(initializeState())
+        val controller = DefaultAboutFragmentController(
+            metrics = context?.metrics,
+            whatsNewViewed = { WhatsNew.userViewedWhatsNew(context!!) },
+            openToCustomTab = ::openLinkInCustomTab,
+            openLibraries = ::openLibrariesPage
+        )
+        val interactor = AboutPageInteractor(controller)
+        aboutView = AboutView(rootView.about_layout, interactor)
+
+        return rootView
+    }
+
+    @ExperimentalCoroutinesApi
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        consumeFrom(store) {
+            aboutView.update(it)
+        }
+    }
+
+    private fun initializeState(): AboutPageState {
+        val items: MutableList<AboutPageItem> = mutableListOf()
+
+        items.add(populateAboutHeader())
```

Comment:
This can be simplified to `listOf(populateAboutHeader()) + populateAboutList()`

Reply:
Thanks! This looks nice.